### DISCUSSION TITLE
Why does Makefile `chmod 777` on the build dir?

### DISCUSSION THREAD

**Discussion by BBaoVanC** [hyprwm/Hyprland#7603] [Q&A]
I didn't want to make an issue because of how many fields I have to fill out, but `git blame` points to 1a0909aa203312b6ccfe8c34ddb6cbec63ca5199. Is there a specific issue that's fixed by chmod-ing the build directory? CMake will automatically set the right (execute) permissions itself, and making the entire dir world readable isn't advisable for security reasons.

**Comment by vaxerski** [comment]
IIRC because back in the day some people would run `sudo make release` and then when they'd try to `cmake --build build` get an error and complain
